### PR TITLE
Fix #179128 timesofindia.indiatimes.com

### LINKED ISSUE #179128: timesofindia.indiatimes.com
### Issue URL (Ads)
 [https://timesofindia.indiatimes.com/city/chennai/separated-by-borders-united-by-heart-pakistan-teen-gets-indians-organ/articleshow/109577501.cms](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftimesofindia.indiatimes.com%2Fcity%2Fchennai%2Fseparated-by-borders-united-by-heart-pakistan-teen-gets-indians-organ%2Farticleshow%2F109577501.cms)
Username: `@xglossyntx`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.46
System version: | Windows 10
Browser: | Firefox
Stealth mode options: | Block WebRTC, <br>Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Self-destructing third-party cookies (2880), <br>Block trackers
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><details><summary><b>Language-specific:</b></summary>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>AdGuard French, <br/>AdGuard Chinese, <br/>EasyList Czech and Slovak, <br/>EasyList Italy, <br/>AdBlockID, <br/>List-KR, <br/>Dandelion Sprout's Nordic Filters, <br/>IndianList</details>

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #179128

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by piquark6046 — DISMISSED** [review]
I cannot reproduce the request with Proton VPN US-WA#48.
Did you do any interaction?

**Comment by xglossyntx** [comment]
I scrolled down the page a bit.

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -2054,6 +2054,7 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||hankyung.com/resource/js/w/view.gtmet.js
 ||discord.com/api/v*/science
 ||fensi.plus/*/tracking/
+||tvid.in/log/pixel
 ||tvid.in/sdk/loader.js
 ||tvid.in/log/logs
```
Comment (piquark6046) [inline SpywareFilter/sections/specific.txt:2059]:
```suggestion
||tvid.in/sdk/loader.js
||tvid.in/log/
```
Please apply the changes.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

**Comment by piquark6046** [comment]
Because simplify the rules into one.

**Comment by xglossyntx** [comment]
Oh, I see what you are trying to do.

**Review by piquark6046 — APPROVED** [review]

**Comment by xglossyntx** [comment]
I was about to change it back.

**Comment by piquark6046** [comment]
Fine. It is still okay.

**Comment by xglossyntx** [comment]
I made a new pull request

**Pull request merged**